core/src/main/scala/org/broadinstitute/dsde/rawls/snapshot/SnapshotService.scala

```diff
@@ -36,37 +40,41 @@ import scala.jdk.CollectionConverters._
 import scala.util.{Failure, Success, Try}
 
 object SnapshotService {
-  def constructor(dataSource: SlickDataSource,
+  def constructor(workspaceRepository: WorkspaceRepository,
```

Comment:
I changed this so I could mock more effectively in the new tests since it was a relatively minor change and I was already changing the constructor.